.github/workflows/reusable-deploy-docs.yaml

```diff
@@ -116,30 +121,35 @@ jobs:
           #   the name of the current repo,
           #   "html_pdf" to indicate producing both html and pdf output, and
           #   the path to the docs template directory in the geoips default branch.
-
           # If we are specifically "template_basic_plugin" repo, then we must use
           # my_package as the package name.  Otherwise the package name matches
           # the plugin name.
           run_on_package=$curr_repo
           if [[ "$curr_repo" == "template_basic_plugin" ]]; then
             run_on_package=my_package
           fi
-          # Note pdf build failing right now - temporarily only produce html so it works
-          # ./geoips_default_branch/docs/build_docs.sh ./current_repo_current_branch $curr_repo html_pdf ./geoips_default_branch/docs
-          ./geoips_default_branch/docs/build_docs.sh ./current_repo_current_branch $run_on_package html_only ./geoips_default_branch/docs
+          export GEOIPS_PACKAGES_DIR="."
+          export GEOIPS_OUTDIRS=""
+          export GEOIPS_TESTDATA_DIR="" # necessary for GeoIPS to be importable for the docs to build. Just has to be set to *something*
+          ln -s ./geoips_default_branch ./geoips # the CLI *requires* the geoips folder to be named "geoips" so soft linking here
```

Comment:
We might want to delete this link when we're done with it.

Reply:
Sure, that works for me

Reply:
Added 👍